### PR TITLE
source{d} does FOSDEM 2018

### PR DESCRIPTION
**Pull request by campoy**
All of the `TODO` will be replaced as soon as I get more info from the speakers.

@estherrgarcia can you provide the picture you took at the dinner?
@bzz can you request the speakers from the SCA devroom that are missing info to provide their social media handlers (if they wish so) and a link to their slides?

Thanks!

### CODE REVIEW

File: content/post/fosdem-2018.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- JavaParser: where should we head?, by [Federico Tomassetti](https://twitter.com/ftomasse) ([🎥](https://www.youtube.com/watch?v=vsn7BzJLBzo&list=PL5Ld68ole7j2H1SnyhDg7DI0fgh0GNuez&index=8) / [📈](TODO))
+- Graph-based analysis of JavaScript repositories, by [Adam Lippai](https://twitter.com/alippai) ([🎥](https://www.youtube.com/watch?v=dYBURFmH9Xk&list=PL5Ld68ole7j2H1SnyhDg7DI0fgh0GNuez&index=9) / [📈](https://fosdem.org/2018/schedule/event/code_graph_based_analysis_of_javascript_repositories/attachments/slides/2508/export/events/attachments/code_graph_based_analysis_of_javascript_repositories/slides/2508/fosdem_js_source_code_analysis_alippai.pdf))
+- DIY Java Static Analysis, by [Nicolas Peru](TODO) ([🎥](https://www.youtube.com/watch?v=h-7TXKMXn0o&list=PL5Ld68ole7j2H1SnyhDg7DI0fgh0GNuez&index=10) / [📈](https://fosdem.org/2018/schedule/event/code_diy_java_static_analysis/attachments/slides/2054/export/events/attachments/code_diy_java_static_analysis/slides/2054/DIY_Java_static_analysis.pdf))
```
Comment (bzz) [inline content/post/fosdem-2018.md]: https://twitter.com/@benzonico

**Comment by campoy** [comment]
Hey @bzz, any news on the missing slides for the SCA room?

File: content/post/fosdem-2018.md [hunk 2]
```diff
@@ -53,29 +53,56 @@ Is a conference really a conference without a fun speakers night? At source{d} w
...
 
+## But there's more
+
+In addition to managing two devrooms and rewarding all the speakers with good food and cold beer,
+two of our employees gave extra talks!
+
+### Tools for large-scale collection and analysis of source code repositories
+
+[Alex Bezzubov](https://twitter.com/seoul_engineer) gave a talk at the HPC, Big Data, and Data Science
+Devroom on the tech stack we develop at [sourced{}](https://sourced.tech).
```
Comment (smacker) [inline content/post/fosdem-2018.md]: sourced{} -> source{d}

File: content/post/fosdem-2018.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+Thanks to all of the speakers in our and other devrooms, to the organizers who did an incredible job,
+to all those that attended our devrooms, and to those that tried but could not get in!
+
+Let's do it all over in 2019!
+
+#### We want you!
+
+Are you interested on speaking at events about Go or Source Code Analysis?
+We are currently organizing events in San Francisco and Madrid, let us know!
```
Comment (mcuadros) [inline content/post/fosdem-2018.md]: and Lisbon

**Comment by rporres** [comment]
As commented, it would be nice to add a note about my talk at configuration management camp. The talk was called "CoreOS Container Linux baremetal provisioning with Terraform, Ignition and Matchbox" and it talked about how we provision baremetal servers here at source{d}. Slides are [here](http://bit.ly/2nCCgoa) and the full abstract http://cfgmgmtcamp.eu/schedule/terraform/CoreOS.html

A couple of references in Twitter (just in case)

* https://twitter.com/sheriffjackson/status/960522495039361024
* https://twitter.com/walter_burns/status/960530295891120129

File: content/post/fosdem-2018.md (hunk 2 above)
Comment (campoy) [reply content/post/fosdem-2018.md]: done

File: content/post/fosdem-2018.md (hunk 3 above)
Comment (campoy) [reply content/post/fosdem-2018.md]: done

**Comment by campoy** [comment]
@rporres let me know if the current description of your talk looks good
I'll be publishing this tomorrow at 8am PST

**Comment by campoy** [comment]
I deployed by mistake, but I think it's ok
I also scheduled a tweet from @sourcedtech at 8AM PST

**Comment by rporres** [comment]
it looks great, @campoy . Thx!

**Pull request merged**